### ISSUE TITLE
"Remember Me" option doesn't work

### ISSUE THREAD

**Issue by Draco18s** [DuxterInc/trendy-community#209]
I don't normally wander into the issue tracker, but this issue has been long standing and does not appear to be officially tracked in any way.  Combine that with the site update changes a few days ago, I can no longer sit on the All Notifications page and refresh each day (as it redirects me to a 404 page--rightfully as I am not signed in), but as I can't make the site *remember who I am* this added significant extra steps to checking for new posts.

**Comment by adamlieb** [comment]
Can you do me a couple quick favors to try and help debug this?
-Clear cache
-Close the tab and hit the domain fresh (so you get a fresh load)
-Log out/login 
-When logging in use the remember me check

Let me know if/when you get locked out (what action caused it, or if it was timed...24 hours?)

I can confirm this is working properly for lots of folks, so I want to understand why it isn't working for you and get it fixed. I think it should be working for you once you get a fresh version of the site.

**Comment by Draco18s** [comment]
Does appear to be working properly today.  Sometime between last night and this morning it went from "definitely not working at all" (a simple page refresh would log me out) to "definitely working, can't seem to break it" this afternoon.

**Comment by adamlieb** [comment]
Sweet (sort of)! Closing the issue, if it happens again please reopen and describe.

let me know what else you find, your issue reports are always super helpful.

**Comment by Draco18s** [comment]
Any time :)

**Comment by adamlieb** [comment]
Can you ping me on Skype?

**Comment by Draco18s** [comment]
Had it go a little weird this morning, but can't replicate.  I was sitting on the All Notifications page, went to News, had it award me 2 influence points (I had 295 yesterday, I now have 297), had to close both (overlapping) popups.  Then I clicked "back" to go back to my notifications page, it didn't.  So I used the multi-jump history dropdown.  I had 2 notifications pages and 2 homepages.  I clicked on the farthest one back.  Which may have been a browser-cached page, as it had the look from a few weeks ago and logged me out, but as soon as I logged in again, it was updated.

Probably a quirk that I got logged out (old, cached page), but the two influence points was the interesting part.
